Queueomatic/Client/Pages/Index.razor.cs

```diff
@@ -21,6 +21,15 @@ private async Task CheckRoomIsValid()
     {
         _isClicked = true;
         _buttonContent = "Processing...";
+        
+        if (_roomId.Length < 6)
+        {
+            Toast.Show("warning", "Wrong input!", 5000);
```

Comment:
I think this should more accurately display what went wrong instead of a general output.

Reply:
_Minimum length of room id is 6 but you entered ***X***._ 
Does it do the work?

Reply:
Skip, *"but you entered X."* and add, "*6 characters.*"